doc/contributing/releases.md

```diff
@@ -704,12 +704,23 @@ the build before moving forward. Use the following list as a baseline:
 
 ### 11. Tag and sign the release commit
 
-Once you have produced builds that you're happy with, create a new tag. By
-waiting until this stage to create tags, you can discard a proposed release if
-something goes wrong or additional commits are required. Once you have created a
-tag and pushed it to GitHub, you _**must not**_ delete and re-tag. If you make
-a mistake after tagging then you'll have to version-bump and start again and
-count that tag/version as lost.
+Once you have produced builds that you're happy with you can either run
+`git node release -S --promote`
```

Comment:
`-S` is the same flag as `git`, and while I agree it's a good idea our examples use it so folks produces signed release commits, it probably makes more sense to skip it in text (for the same reason you would say `git commit`, not `git commit -S`)
```suggestion
`git node release --promote`
```